src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java

```diff
@@ -19,29 +19,39 @@
 import org.openqa.selenium.remote.CapabilityType;
 
 /**
- * The list of Android-specific capabilities.
- * Read:
- * https://github.com/appium/appium/blob/1.5/docs/en/writing-running-appium/caps.md#android-only
+ * The list of Android-specific capabilities. <br>
+ * Read: <br>
+ * <a href="https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/caps.md#android-only">
+ * https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/caps.md#android-only</a>
  */
 public interface AndroidMobileCapabilityType extends CapabilityType {
+
     /**
      * Activity name for the Android activity you want to launch from your package.
-     * This often needs to be preceded by a . (e.g., .MainActivity instead of MainActivity).
+     * This often needs to be preceded by a {@code .} (e.g., {@code .MainActivity}
+     * instead of {@code MainActivity}). By default this capability is received from the package
+     * manifest (action: android.intent.action.MAIN , category: android.intent.category.LAUNCHER)
      */
     String APP_ACTIVITY = "appActivity";
 
     /**
-     * Java package of the Android app you want to run.
+     * Java package of the Android app you want to run. By default this capability is received
+     * from the package manifest ({@literal @}package attribute value)
      */
     String APP_PACKAGE = "appPackage";
 
     /**
-     * Activity name for the Android activity you want to wait for.
+     * Activity name/names, comma separated, for the Android activity you want to wait for.
+     * By default the value of this capability is the same as for {@code appActivity}.
+     * You must set it to the very first focused application activity name in case it is different
+     * from the one which is set as {@code appActivity} if your capability has {@code appActivity}
+     * and {@code appPackage}.
      */
     String APP_WAIT_ACTIVITY = "appWaitActivity";
 
     /**
      * Java package of the Android app you want to wait for.
+     * By default the value of this capability is the same as for {@code appActivity}
```

Comment:
We can also mention, that this cap can include wildcards (`*`)

Reply:
done.